### ISSUE TITLE
lambda operator not supported in java 8 

### ISSUE THREAD

**Issue by GulraizAsmat** [Codigami/CFAlertDialog#34]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/Codigami/CFAlertDialog/blob/develop/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/Codigami/CFAlertDialog/blob/develop/README.md)
* [ ] I have searched for a similar issue in the [project](https://github.com/Codigami/CFAlertDialog/issues) and found none

### Issue Info

 Info                          | Value                               |
-------------------------------|-------------------------------------|
 Platform Name                 | e.g. Android
 Platform Version              | e.g. 6.0
 CFAlertDialog Version         | e.g. 2.1.1
 Integration Method            | e.g. gradle / manually
 Xcode Version                 | e.g. Android Studio 2.3
 Repro rate                    | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj       | e.g. does it happen with our demo project?
 Demo project link             | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by rahul-s** [comment]
@GulraizAsmat  Can you please explain what issue you are facing..?

**Comment by AkinKeskinbas** [comment]
java.lang.NullPointerException: 
  at com.akin.test.Anasayfa.lambda$showStartDialog$1$Anasayfa (Anasayfa.java:346)
  at com.akin.test.-$$Lambda$Anasayfa$eYk8QGBUd2jY2YmTpQIEE9uJNPY.onClick (lambda)
  at com.crowdfire.cfalertdialog.CFAlertDialog$6.onClick (CFAlertDialog.java:564)
  at android.view.View.performClick (View.java:5205)
  at android.view.View$PerformClick.run (View.java:21164)
  at android.os.Handler.handleCallback (Handler.java:739)
  at android.os.Handler.dispatchMessage (Handler.java:95)
  at android.os.Looper.loop (Looper.java:148)
  at android.app.ActivityThread.main (ActivityThread.java:5417)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:726)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:616)
